<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assertViewer</class>
 <widget class="QWidget" name="assertViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2633</width>
    <height>1322</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1761</width>
    <height>984</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>assertViewer - ${title}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="PyDMTabWidget" name="PyDMTabWidget_main">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alarmChannels" stdset="0">
        <stringlist>
         <string></string>
        </stringlist>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PlotWidget" name="graphicsView_3"/>
     </item>
     <item row="0" column="0">
      <widget class="PlotWidget" name="graphicsView_1"/>
     </item>
     <item row="1" column="0">
      <widget class="PlotWidget" name="graphicsView_2"/>
     </item>
     <item row="1" column="1">
      <widget class="PlotWidget" name="graphicsView_4"/>
     </item>
     <item row="1" column="2">
      <widget class="PyDMTabWidget" name="PyDMTabWidget_config">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alarmChannels" stdset="0">
        <stringlist>
         <string></string>
        </stringlist>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Configuration</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="4" column="1">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Configure beam below:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="PyDMCheckbox" name="PyDMCheckbox_feature1">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Feature 1</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="PyDMCheckbox" name="PyDMCheckbox_feature3">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Feature 3</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="PyDMCheckbox" name="PyDMCheckbox_feature2">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Feature 2</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMTabWidget</class>
   <extends>QTabWidget</extends>
   <header>pydm.widgets.tab_bar</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PyDMCheckbox</class>
   <extends>QCheckBox</extends>
   <header>pydm.widgets.checkbox</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
